### PR TITLE
[kbss-cvut/record-manager-ui#71] Reduce the size of PatientRecordDto data

### PR DESCRIPTION
**Pull request by ledsoft**
- Also update code to the new OWL2Java constant naming strategy.
- Update dependencies to work with the latest JOPA 2.0.0-SNAPSHOT build.

kbss-cvut/record-manager-ui#71.

### CODE REVIEW

**Comment by ledsoft** [comment]
Note that the changes regarding kbss-cvut/record-manager-ui#71 are backwards compatible with the frontend, so the PR can be merged without waiting for frontend implementation.

**Comment by ledsoft** [comment]
@blcham

File: src/main/java/cz/cvut/kbss/study/dto/PatientRecordDto.java [hunk 1]
```diff
@@ -21,17 +22,17 @@ public class PatientRecordDto extends AbstractEntity implements HasOwlKey {
     private String localName;
 
     @ParticipationConstraints(nonEmpty = true)
-    @OWLObjectProperty(iri = Vocabulary.s_p_has_author, fetch = FetchType.EAGER)
-    private User author;
+    @OWLObjectProperty(iri = Vocabulary.s_p_has_author)
+    private URI author;
```
[the remark this answers is not in the document]
Comment (blcham) [reply src/main/java/cz/cvut/kbss/study/dto/PatientRecordDto.java:26]: image

**Review by blcham — APPROVED** [review]
LGTM, just have a look at my comment.

**Pull request merged**